docs/settings/settings.md

```diff
@@ -309,7 +401,7 @@ A Rest API can be activated in order to:
 - Get synapse's detail
 - Run a synapse
 
-For the complete API ref see the [REST API documentation](rest_api.md)
```

Comment:
In the genrate doc the list after : "A Rest API can be activated in order to:" is not properly displayed

Reply:
Plus, the list is not up to date: 
it should also include : 

| Method | URL                               | Action                             |
|--------|-----------------------------------|------------------------------------|
| GET    | /settings/deaf                    | Get the current deaf status        |
| POST   | /settings/deaf                    | Switch the deaf status             |
| GET    | /settings/mute                    | Get the current mute status        |
| POST   | /settings/mute                    | Switch the mute status             |
| GET    | /settings/energy_threshold        | Get the current energy_threshold   |
| POST   | /settings/energy_threshold        | Update the energy_threshold value  |
| GET    | /settings/ambient_noise_second    | Get the ambient_noise_second       |
| POST   | /settings/ambient_noise_second    | Update the ambient_noise_second    |
| GET    | /settings/hooks                   | Get the current hooks              |
| POST   | /settings/hooks                   | Update the hooks list              |
| GET    | /settings/variables               | Get the variables list             |
| POST   | /settings/variables               | Update the variables list          |
| GET    | /settings/default_tts             | Get current tts                    |
| POST   | /settings/default_tts             | Update current tts                 |
| GET    | /settings/default_stt             | Get current stt                    |
| POST   | /settings/default_stt             | Update current stt                 |
| GET    | /settings/default_player          | Get the current player             |
| POST   | /settings/default_player          | Update current player              |
| GET    | /settings/default_trigger         | Get the current trigger            |
| POST   | /settings/default_trigger         | Update the current trigger         |

Reply:
done